certified-connectors/Transform2All/apiProperties.json

```diff
@@ -0,0 +1,47 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "<Please add your clientID here>",
+          "customParameters": {
+            "loginUri": {
+              "value": "https://login.windows.net"
+            },
+            "resourceUri": {
+              "value": "api://transform2all.gaccloud.com"
+            },
+            "tenantId": {
+              "value": "common"
+            }
+          },
+          "identityProvider": "aad",
+          "properties": {
+            "AzureActiveDirectoryResourceId": "api://transform2all.gaccloud.com",
+            "IsFirstParty": "False"
+          },
+          "redirectMode": "Global",
+          "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+          "scopes": []
+        },
+        "type": "oauthSetting"
+      },
+      "token:TenantId": {
+        "metadata": {
+          "sourceType": "AzureActiveDirectoryTenant"
+        },
+        "type": "string",
+        "uiDefinition": {
+          "constraints": {
+            "hidden": "true",
+            "required": "false"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#FA9901",
```

Comment:
@Joerio The iconBrandColor should be "#052D40" as per the latest submission you made and approved in the certification portal. Can you please update it to "#052D40"

Reply:
I updated the icon color and committed it again.